src/test/java/com/google/ehub/servlets/ItemPageServletTest.java

```diff
@@ -116,13 +118,81 @@ public void doGetReturnsItemPageData() throws IOException {
     verify(printWriter).println(new Gson().toJson(itemData));
   }
 
+  @Test
+  public void doGetReturnsItemPageData_differentUser() throws IOException {
+    EntertainmentItem selectedItem = new EntertainmentItem.Builder()
+                                         .setTitle(TITLE)
+                                         .setDescription(DESCRIPTION)
+                                         .setImageUrl(IMAGE_URL)
+                                         .setReleaseDate(RELEASE_DATE)
+                                         .setRuntime(RUNTIME)
+                                         .setGenre(GENRE)
+                                         .setDirectors(DIRECTORS)
+                                         .setWriters(WRITERS)
+                                         .setActors(ACTORS)
+                                         .setOmdbId(OMDB_ID)
+                                         .build();
+    Key itemId = EntertainmentItemDatastore.getInstance().addItemToDatastore(selectedItem);
+    when(request.getParameter("itemId")).thenReturn(itemId.getId() + "");
+    when(response.getWriter()).thenReturn(printWriter);
+    Key commentId = commentDataManager.addItemComment(itemId.getId(), COMMENT, TIMESTAMP, ALT_EMAIL);
+    servlet.doGet(request, response);
+    CommentData comment = new CommentData(
+        itemId.getId(), COMMENT, TIMESTAMP, USERNAME, commentId.getId(), belongsToUser);
+    Optional<EntertainmentItem> expectedItem =
+        EntertainmentItemDatastore.getInstance().queryItem(itemId.getId());
+    ItemPageData itemData = new ItemPageData(expectedItem.get(), Lists.newArrayList(comment));
+    CommentData test_comment = commentDataManager.retrieveComments(12345).get(0);
+    verify(response).setContentType(JSON_CONTENT_TYPE);
+    verify(printWriter).println(new Gson().toJson(itemData));
+    Assert.assertEquals(false, test_comment.belongsToUser);
```

Comment:
Adding some spaces between lines would probably improve readability. In the case of this test I think you could add an empty line between sections that setup the test, call the function to be tested, and assert that the results are valid.
Kind of like this:
```suggestion
    Key itemId = EntertainmentItemDatastore.getInstance().addItemToDatastore(selectedItem);
    when(request.getParameter("itemId")).thenReturn(itemId.getId() + "");
    when(response.getWriter()).thenReturn(printWriter);
     Key commentId = commentDataManager.addItemComment(itemId.getId(), COMMENT, TIMESTAMP, ALT_EMAIL);
    
    servlet.doGet(request, response);
    
    CommentData comment = new CommentData(
        itemId.getId(), COMMENT, TIMESTAMP, USERNAME, commentId.getId(), belongsToUser);
    Optional<EntertainmentItem> expectedItem =
        EntertainmentItemDatastore.getInstance().queryItem(itemId.getId());
    ItemPageData itemData = new ItemPageData(expectedItem.get(), Lists.newArrayList(comment));
    CommentData test_comment = commentDataManager.retrieveComments(12345).get(0);
    
    verify(response).setContentType(JSON_CONTENT_TYPE);
    verify(printWriter).println(new Gson().toJson(itemData));
    Assert.assertEquals(false, test_comment.belongsToUser);
```